src/app/starredentries/starredentries.component.html

```diff
@@ -81,12 +82,42 @@
           </div>
         </mat-tab>
 
-        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool">
+        <mat-tab>
+          <ng-template mat-tab-label>
+            <div fxLayoutGap="0.5rem">
+              <img class="notebook-icon" src="../../../assets/svg/sub-nav/notebooks.svg" alt="notebook logo" />
+              <span>Notebooks</span>
+              <span class="tab-display" data-cy="starred-notebooks-count">{{ starredNotebooks?.length }}</span>
+            </div>
+          </ng-template>
+          <div>
+            <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredNotebooks?.length === 0">
+              <mat-icon>warning</mat-icon>
+              You have no starred notebooks.
+            </mat-card>
+            <div fxLayout="column" fxLayoutGap="1rem" class="mb-2 mx-1">
+              <mat-card fxFlex *ngFor="let notebook of starredNotebooks; let last = last" class="m-1 pb-3">
+                <ng-container
+                  *ngTemplateOutlet="workflowSummary; context: { workflow: notebook, isAppTool: false, isNotebook: true }"
+                ></ng-container>
+              </mat-card>
+            </div>
+          </div>
+        </mat-tab>
+
+        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool" let-isNotebook="isNotebook">
           <div fxLayout="row" fxLayoutGap="10px" fxLayoutAlign="space-between">
             <a [routerLink]="(isAppTool ? entryType.AppTool : entryType.BioWorkflow) | routerLink: workflow" class="no-underline">
               <mat-card-header>
                 <mat-card-title fxLayoutAlign="space-between" fxLayoutGap="1rem">
                   <img
+                    *ngIf="isNotebook"
+                    src="../../../assets/svg/sub-nav/notebooks.svg"
+                    alt="{{ isAppTool ? 'tool' : 'workflow' }} logo"
```

Comment:
You can hard-code `"notebook logo"` here.

Reply:
The Entry responses from the webservice now contain an `entryTypeMetadata` property, which has sub-properties `term` and `termPlural` that contain the singular and plural words used to refer to that kind of entry.  So, if we make our image names consistent (either singular or plural, depending on context), we could coalesce all the cases into a single passage, like:
```
<img
  src="../../../assets/svg/sub-nav/{{ workflow.entryTypeMetadata.termPlural }}.svg"
  alt="{{ workflow.entryTypeMetadata.term }} logo"
/>
```
We might even put this in a pipe someday:
```
{{ workflow | subNavImg }}
```

Sometime (not now), we should refactor the `routerLink` pipe similarly.  Now that the workflow object contains its type and other necessary information, the entryType argument is superfluous.